src/operator/tensor/elemwise_binary_op_basic.cc

```diff
@@ -166,6 +168,8 @@ The storage type of ``elemwise_sub`` output depends on storage types of inputs
 
    - elemwise_sub(row_sparse, row_sparse) = row_sparse
    - elemwise_sub(csr, csr) = csr
+   - elemwise_sub(dns, csr) = dns
```

Comment:
We never use "dns" API doc. Either "default" or "dense" is fine

Reply:
Done